third_party/move/move-compiler-v2/tests/checking/typing/type_variable_join_single_unpack.exp

```diff
@@ -9,11 +9,11 @@ module 0x8675309::M {
     }
     private fun t0() {
         {
-          let M::Box<?2>{ f1: f1: u64, f2: f2: u64 } = M::new<u64>();
+          let M::Box<?1>{ f1: f1: u64, f2: f2: u64 } = M::new<u64>();
```

Comment:
These `?1` types are pretty poor.  I've filed an issue https://github.com/aptos-labs/aptos-core/issues/10994 to fix this output to be more readable, and hopefully deterministic.  (But don't try to fix it here.)

Reply:
@brmataptos I don't quite get why you are complaining about error messages which this PR doesn't change at all, it just adds new test cases which produce new errors. Can we please focus on the PR at hand? If you have concerns about error messages, can you open issues instead?